### PR TITLE
Upgrading to API V5 

### PR DESCRIPTION
**Pull request by franciscofsales**
Please proceed with care. Its got a lot of changes as the API also changed dramatically.

I have put up what I think its a good code base to work from. 

Most stuff is already implemented and working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,28 +18,25 @@ Require the library and initialialize it with your account key:
 var Bing = require('node-bing-api')({ accKey: "your-account-key" });
 ```
 
-#### Web Search:
+#### Web Search (same as Composite):
 ```js
 Bing.web("Pizza", {
     top: 10,  // Number of results (max 50)
-    skip: 3,   // Skip first 3 results
-    options: ['DisableLocationDetection', 'EnableHighlighting']
```
Comment (goferito) [inline README.md]: There are no search options any more?

Comment (franciscofsales) [inline README.md]: and also it was merged with Combined

File: test/integration.js [hunk 2]
```diff
@@ -114,10 +107,9 @@ describe("Bing News", function () {
 
   it('finds news with specific options', function (done) {
 
-    Bing.news('ps4',
+    Bing.news('microsoft',
```
Comment (goferito) [inline test/integration.js]: c'mon.. this removes the fun :laughing:

File: test/integration.js [hunk 3]
```diff
@@ -41,39 +40,38 @@ describe("Bing Web", function () {
...
-      body.d.results.should.have.length(5);
+      body.webPages.value.should.have.length(5);
 
       done();
     });
   });
 
-  it('finds ukrainian results', function(done){
-    Bing.web('"Sony Xperia Z3" огляди тест',
+  it('finds russian results', function(done){
```
Comment (goferito) [inline test/integration.js]: haha what an insinuation :laughing:

Comment (franciscofsales) [inline test/integration.js]: ukranian is no longer supported :p I guess microsoft is capitalist, so they may assume Russia capitalized on Ukraine, kind of weird on them

**Comment by goferito** [comment]
Ok, I don't have much time to properly test it, so I will just trust you on this one. I still think the docs could be improved a bit by better displaying the options though.

But thanks a lot for the contribution :+1:

**Comment by franciscofsales** [comment]
@goferito I will work improve the docs. You have my word I will. Well it was not working, now most stuff is working :)

**Comment by goferito** [comment]
I'll publish it now as v3 on npm. Thanks again.

**Comment by goferito** [comment]
closes #24

**Pull request merged**